client/src/components/csv/CSVWizard.tsx

```diff
@@ -0,0 +1,150 @@
+import React from "react";
+import {Button, Container, Dimmer, Header, Loader, Segment, Step} from 'semantic-ui-react';
+import UploadStep from './CSVUpload';
+import ReviewStep from './CSVReview';
+import { ItemComplete } from "../item/ItemEdit";
+import { withToastManager } from "react-toast-notifications";
+import { Redirect } from "react-router-dom";
+
+interface CSVWizardProps {
+    toastManager: any
+}
+
+interface CSVWizardState {
+    wizardStep: number,
+    inventory: ItemComplete[], 
+    isStepComplete: boolean,
+    isSubmitting: boolean,
+    isComplete: boolean
+}
+
+interface StepInterface {
+    [key: string]: any,
+    key: string,
+    icon: string,
+    title: string,
+    description?: string,
+    stepDiv?: React.ReactElement,
+    active?: boolean,
+    disabled?: boolean
+}
+
+class CSVWizard extends React.Component<CSVWizardProps, CSVWizardState> {
+    constructor(props: any) {
+        super(props);
+        this.state = {
+            wizardStep: 0,
+            inventory: [],
+            isStepComplete: false,
+            isSubmitting: false,
+            isComplete: false
+        };
+    }
+
+    nextStep = () => {
+        const { wizardStep } = this.state; 
+        this.setState({ wizardStep: wizardStep+1,
+                        isStepComplete: false });
+    }
+
+    setInventory = (inventory: ItemComplete[]) => {
+        this.setState({inventory, isStepComplete: true});
+    }
+
+    uploadInventory = () => {
+        this.setState({isSubmitting: true});
+        window.alert("Unimplemented");
+        window.setTimeout(this.onInventorySubmitted, 2000);
+    }
+
+    onInventorySubmitted = () => {
+        const { toastManager } = this.props;
+        this.setState({isSubmitting: false});
+        toastManager.add('CSV Submitted! Redirecting', {
+            appearance: 'success',
+            autoDismiss: true,
+            placement: "top-center"
+        });
+        window.setTimeout(() => {
+            this.setState({isComplete: true});
+        }, 2000); // Redirect, since you should be done with the wizard
```

Comment:
not sure the 2 second delay on this redirect is necessary so long as the toast still appears.

Reply:
single page app toasts amirite